packages/core/src/transport/eventBridge.ts

```diff
@@ -1,25 +1,34 @@
-import { isExperimentalFeatureEnabled } from '..'
+import { includes, isExperimentalFeatureEnabled } from '..'
 
 export interface BrowserWindow extends Window {
   DatadogEventBridge?: DatadogEventBridge
 }
 
 export interface DatadogEventBridge {
+  getAllowedWebViewHosts(): string
   send(msg: string): void
 }
 
 export function getEventBridge<T, E>() {
-  const datadogEventBridge = getEventBridgeGlobal()
+  const eventBridgeGlobal = getEventBridgeGlobal()
+
+  if (!eventBridgeGlobal) {
+    return
+  }
 
   return {
+    getAllowedWebViewHosts() {
+      return JSON.parse(eventBridgeGlobal.getAllowedWebViewHosts()) as string[]
+    },
     send(eventType: T, event: E) {
-      datadogEventBridge?.send(JSON.stringify({ eventType, event }))
+      eventBridgeGlobal.send(JSON.stringify({ eventType, event }))
     },
   }
 }
 
 export function isEventBridgePresent(): boolean {
-  return !!getEventBridgeGlobal()
+  const bridge = getEventBridge()
+  return !!bridge && includes(bridge.getAllowedWebViewHosts(), window.location.hostname)
```

Comment:
we don't check only if the bridge is present now.
what about renaming the function to something like `canUseEventBridge`?

Reply:
yes sure